s4e-web/cypress/page-objects/map/map-products.po.ts

```diff
@@ -1,42 +1,87 @@
 import { Core } from '../core.po';
 
 export class MapProducts extends Core {
-   
-    static pageObject = {
-
-      getAllProductsInAllCategories: () => cy.get('[data-e2e="product-list"] [data-e2e="products__item"]'),
-      getProductNavbar: () => cy.get('[data-e2e="product-list"]'),
-      getProductsNameBtns: () => cy.get('[data-e2e="product-list"] [data-e2e="picker-item-label"]'),
-    }
-
-    static selectProductBy(partialName: string) {
-        MapProducts
-          .pageObject
-          .getProductsNameBtns()
-          .contains(partialName)
-          .should("be.visible")
-          .click();
-        return MapProducts;
-      }
-    
-      static selectNthProduct(number: number) {
-        MapProducts
-          .pageObject
-          .getProductsNameBtns()
-          .eq(number)
-          .should("be.visible")
-          .click();
-        return MapProducts;
-      }
-    
-      static productsCountShouldBe(count: number) {
-        MapProducts
-          .pageObject
-          .getAllProductsInAllCategories()
-          .should('have.length', count);
-    
-        return MapProducts;
-      }
+
+  static readonly pageObject = {
+    getAllProductsInAllCategories: () => cy.get('[data-e2e="product-list"] [data-e2e="products__item"]'),
+    getProductNavbar: () => cy.get('[data-e2e="product-list"]'),
+    getProductsNameBtn: () => cy.get('[data-e2e="product-list"] [data-e2e="picker-item-label"]'),
+    getLegend: () => cy.get('[data-e2e="legend__chart"]'),
+    getOnLiveBtn: () => cy.get('[data-e2e="btn-live"]'),
+  };
+
+  static selectProductByName(partialName: string) {
+    cy.route('GET', '/api/v1/products/*').as('turnOnProduct')
     
+    MapProducts
+      .pageObject
+      .getProductsNameBtn()
+      .contains(partialName)
+      .should("be.visible")
+      .click();
+
+    cy.wait('@turnOnProduct');
+    cy.get('.products__visibility__spinner').should('not.exist');
+
+    MapProducts
+      .pageObject
+      .getProductsNameBtn()
+      .contains(partialName)
+      .should("have.class", "active");
+
+    return MapProducts;
+  }
+
+  static selectNthProduct(number: number) {
+    MapProducts
+      .pageObject
+      .getProductsNameBtn()
+      .eq(number)
+      .should("be.visible")
+      .click();
+    return MapProducts;
+  }
+
+  static productsCountShouldBe(count: number) {
+    MapProducts
+      .pageObject
+      .getAllProductsInAllCategories()
+      .should('have.length', count);
+
+    return MapProducts;
+  }
+
+  static legendShouldBeVisible() {
+    MapProducts
+      .pageObject
+      .getLegend()
+      .should("be.visible");
+
+    return MapProducts;
+  }
+
+  static turnOnOnLiveView() {
+    cy.route('GET', "/api/v1/products/*/scenes/most-recent?{*,*/*}").as('loadResentScene');
```

Comment:
Are you sure that `{*}` isn't enough?

Reply:
Yess, If  I use {*}, tests doesn't work locally